README.md

```diff
@@ -1,5 +1,22 @@
 # <img src="./docs/assets/logo-alt.png" height="60px" align="center" alt="Flexpa logo"> Flexpa Operating System
 
+## Contributing to the OS
```

Comment:
Maybe a step 0 where you seek feedback where appropriate from other Flexpals before creating a PR.

Reply:
good call! I just added one - what do you think of this wording?